lib/mollie/api/object/payment.rb

```diff
@@ -15,50 +15,103 @@ class Payment < Base
         RECURRINGTYPE_RECURRING = "recurring"
 
         attr_accessor :id,
-                      :status,
                       :mode,
-                      :amount,
-                      :description,
-                      :method,
                       :created_datetime,
+                      :status,
                       :paid_datetime,
-                      :expired_datetime,
                       :cancelled_datetime,
+                      :expired_datetime,
+                      :expiry_period,
+                      :amount,
+                      :amount_refunded,
+                      :amount_remaining,
+                      :description,
+                      :method,
+                      :metadata,
+                      :locale,
+                      :profile_id,
+                      :settlement_id,
                       :customer_id,
                       :recurring_type,
                       :mandate_id,
                       :subscription_id,
-                      :settlement_id,
-                      :metadata,
-                      :details,
-                      :links
+                      :links,
+                      :details
 
         def open?
```

Comment:
In the subscriptions object, you created new methods to check the status, and you prefixed the name of these methods with `status_`, like `status_open?`. I don't really prefer one option over the other, but I would like to keep it consistent. This 2.0 release would be the perfect moment to changing the names of these methods if we want to.